### DISCUSSION TITLE
Type '{ name: string; areas: string; }' is not assignable to type 'Map'

### DISCUSSION THREAD

**Discussion by jasymons** [img-mapper/img-mapper#61] [Q&A] [answered]
**Describe the bug**
There's an error on the map parameter of the ImageMapper component: "Type '{ name: string; areas: string; }' is not assignable to type 'Map'. Types of property 'areas' are incompatible.  Type 'string' is not assignable to type 'MapAreas[]'."

**To Reproduce**
Use <ImageMapper src{URL} map={MAP} /> with the example JSON from the npm page: https://raw.githubusercontent.com/img-mapper/react-docs/master/src/assets/example.json

**Error Stack**
Type '{ name: string; areas: string; }' is not assignable to type 'Map'. Types of property 'areas' are incompatible.  Type 'string' is not assignable to type 'MapAreas[]'.

**Expected behavior**
Map should work

**Screenshots**
![Screenshot 2023-02-28 at 15 51 22](https://user-images.githubusercontent.com/59198762/221890547-8dfec99c-7d69-4e6f-bd1a-22b0edf8924a.png)

**Desktop (please complete the following information):**
 - OS: macOS Ventura
 - Browser: Chrome
 - Version: 110

**Comment by NishargShah** [comment, marked as answer]
Hello @jasymons, `areas` is `JSON` and you need to copy it from the `example.json` and paste it there

Sample Codesandbox: https://codesandbox.io/s/spring-monad-vyfltk?file=/src/App.js

Please check the docs: https://img-mapper.github.io/react-docs/?path=/story/examples-simple--simple

![image](https://user-images.githubusercontent.com/35843261/221896736-a291ed28-2715-4918-a1f9-41b291de4e2a.png)
